src/app/pages/OnDemandRadioPage/index.test.jsx

```diff
@@ -83,7 +84,7 @@ const getAvailableEpisode = assocPath(
   'available',
 );
 
-const pageType = 'media';
+const pageType = MEDIA_PAGE;
```

Comment:
In other files, you've pruned this variable, which I agree with! I think you could do the same here for consistency and assign MEDIA_PAGE to properties directly.

Reply:
Yeah I agree, this wasn't pruned because higher up in the file it was being used directly and I wasn't sure if the `Import as` override would let you use the original constant